main.go

```diff
@@ -119,11 +121,13 @@ func main() {
 
 	setupLogger.V(1).Info("Determining kernel labeling method", KernelLabelingMethodEnvVar, kernelLabelingMethod)
 
+	filterAPI := filter.New(client, mgr.GetLogger())
```

Comment:
```suggestion
	var filterAPI filter.Filter = filter.New(client, mgr.GetLogger())
```

Reply:
why do you want to define it explicitly? We are just passing all the APIs to the manager/other functions, without actually using them

Reply:
This one is a nit. I am OK with it either way.